source/detail/limits.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2014-2021 Thomas Fussell
```

Comment:
What should we do with the copyright?
At least updating the year if creating a new file?
Changing Thomas Fussell by XLNT Community?
Or should we keep Thomas Fussell and just add an extra copyright line? Or should we only keep his name in existing files?

Also AUTHORS may need some update in the future, not sure if we need to add all are names or just XLNT Community and refer to the git committers. The list in authors are currently really small compared to the numbers of people that actually contributed at least a single commit.

Reply:
I think the standard procedure is to keep the original author and add an extra copyright line, yes. But this is without having researched the actual recommendations, but just based on what I saw in other projects. The reason why I left it like this - for now - is because [my own XLNT repository](https://github.com/doomlaur/xlnt) was forked from the original one, and [Pull Request #736](https://github.com/tfussell/xlnt/pull/736) should still be merged - if that will ever happen - to the original repository. In other words, in my (slightly unresearched) opinion, it is correct for my PR to still contain Thomas Fussell's copyright. Of course, as I said, I'm not an expert on this and we should probably research it better before deciding on how to go forward.

As for new Pull Requests, I agree, we should modify the copyright, in my opinion by adding an additional line. TODO: research this topic.

Reply:
Ok, I agree that on existing files keeping Thomas Fussell is a good idea.

Let's merge this PR.